### PR TITLE
Tree Picker: Open item in Infinity Editor

### PR DESCRIPTION
**Pull request by leekelleher**
Wires up the "Show open button" feature in Multinode Treepicker UI.

The Infinity Editor for Documents works, but the Media and Members aren't ready yet, but the plumbing is there for when it is.

### CODE REVIEW

File: src/packages/members/members/components/input-member/input-member.element.ts [hunk 1]
```diff
@@ -143,16 +155,29 @@ export class UmbInputMemberElement extends FormControlMixin(UmbLitElement) {
...
 				<!-- TODO: implement is deleted <uui-tag size="s" slot="tag" color="danger">Deleted</uui-tag> -->
 				<uui-action-bar slot="actions">
+					${this.#renderOpenButton(item)}
 					<uui-button
 						@click=${() => this._requestRemoveItem(item)}
-						label="Remove member ${item.name}"
-						>Remove</uui-button
-					>
+						label="Remove member ${item.name}">
+						Remove
```
Comment (iOvergaard) [inline src/packages/members/members/components/input-member/input-member.element.ts:162]: would you add localization here, please?

Comment (leekelleher) [reply src/packages/members/members/components/input-member/input-member.element.ts]: Localized the label in commit 7e453b3d92ce23776f6d98247a12e9233f9a45d9.

**Review by nielslyngsoe — APPROVED** [review]
Looks good :-)

**Review by nielslyngsoe — APPROVED** [review]
looks good

**Pull request merged**